internal/generator/fluentd/conf_test.go

```diff
@@ -480,21 +480,45 @@ var _ = Describe("Testing Complete Config Generation", func() {
     undefined_dot_replace_char 'UNUSED'
     undefined_max_num_fields '-1'
     process_kubernetes_events 'false'
+    <level>
+      name critical
+      match 'level=critical|Value:critical|"level":"critical"'
+    </level>
+    <level>
+      name error
+      match 'level=error|Value:error|"level":"error"'
+    </level>
     <level>
       name warn
-      match 'Warning|WARN|W[0-9]+|level=warn|Value:warn|"level":"warn"'
+      match 'level=warn|Value:warn|"level":"warn"'
```

Comment:
@jcantrill: From the Jira ticket requirement was to extract level from from logs if value is explicitly present  For example **"Error 02:47:01.619035 This looks like level=warn"**   level extractor should extract "warn" level not "error".  If we segment rules like in the current PR they should be matched in the order they are defined. It will allows to trigger rules where log message have explicit level.  This idea worked when  I tested this using fluentd rewrite_tag_filter where I defined `<rule>`  tag like 

>     <rule>
>       key message
>       pattern /^.*(?<level>\"message\":.*(level=critical|Value:critical|\"level\":\"critical\")).*$/
>       tag level.critical.${tag}
>     </rule>

But these are not working when I modified current rules ingress.go. After debugging `ViaqDataModel` plugin  I found that all rules are combined into one regular expression so rule ordering will not matter. Matcher matches whichever token matches first and it returns the matched level. 

If we want to implement we would need to change `ViaqDataModel` plugin code where match happens sequentially. Please advise.

Reply:
> example "Error 02:47:01.619035 This looks like level=warn" level extractor should extract "warn" level not "error". If

I believe your summary is incorrect; this matches exactly how it was intended.  "Error" is the primary level that this was logged because in most log specs the level is the thing that comes first.  I wold encourage you to review the samples of log messages that you extracted and you will note they all fit the pattern as described.  It's typically something like: <error level> <date> <messages>  or the level and date are swapped